rpa-builder/6.3/modules/ROOT/pages/toolbox-variable-handling.adoc

```diff
@@ -7,15 +7,18 @@ Variable Handling enables you to perform multiple actions like mathematical oper
 * xref:toolbox-variable-handling-activity-parameters.adoc[Activity Parameters]
 +
 The Action Step Activity Parameters (AP) enables you to map variables that you can use not only in the Workflow, but also in the activities of the business process and gateways.

```

Comment:
subject-verb agreement? 
(parameters enable)

Is there a way to flip this to make the user task first? "Map variables..."